TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function-test-01.xml

```diff
@@ -63,7 +63,7 @@
 
     <testCase id="008">
         <description>return null for singleton null</description>
-        <resultNode name="decision008" type="decision">
+        <resultNode errorResult="true" name="decision008" type="decision">
```

Comment:
Case:
https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function.dmn#L70

DMN Spec:

> list of Boolean items or argument list of one or more Boolean items

Concur in the result, but I believe is missing the errorFlag to clarify this is an error case